javascript/javascript2/week1/find-SmallerWord.js

```diff
@@ -0,0 +1,23 @@
+let danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function findSmallerWord(danishWords)
+{
+    let wordLength;
+    let wordIndex;
+    let minWordLength = 256;  // choose a max number
```

Comment:
You can set the `minWordLength` to the first item in the array, then you don't have to specifically set a min word length.
```javascript
let minWordLength = danishWords[0]
```